backend/src/openbeheer/zaakobjecttypen/api/views.py

```diff
@@ -54,6 +72,28 @@ class ZaakObjectTypeListView(
     endpoint_path = "zaakobjecttypen"
 
 
+def expand_zaakobjecttype(
+    client: APIClient, zaakobjecttypen: Iterable[ZaakObjectType]
+) -> Iterable[ObjectType | None]:
+    # We are in the detail endpoint, so there is only one ZaakObjectType
+    zaakobjecttype = list(zaakobjecttypen)[0]
+
+    with objecttypen_client() as ot_client:
+        objecttype_uuid = furl(zaakobjecttype.objecttype).path.segments[-1]
+        objecttype = fetch_one(ot_client, f"objecttypes/{objecttype_uuid}", ObjectType)
+
+    try:
+        objecttype = fetch_one(ot_client, f"objecttypes/{objecttype_uuid}", ObjectType)
```

Comment:
I think this should fetch once?

```suggestion
    try:
        with objecttypen_client() as ot_client:
            objecttype_uuid = furl(zaakobjecttype.objecttype).path.segments[-1]
            objecttype = fetch_one(ot_client, f"objecttypes/{objecttype_uuid}", ObjectType)
```

Reply:
:facepalm: